examples/file-sharing/Cargo.toml

```diff
@@ -6,12 +6,16 @@ publish = false
 license = "MIT"
 
 [dependencies]
+libp2p-request-response = { workspace = true, features = ["cbor"]  }
```

Comment:
Ah this is a bit unfortunate. Can you add a `cbor` feature to the `libp2p` crate and forward it to `libp2p-request-response`? I'd like to avoid users having to declare dependencies on these crates. It is a mess to keep the versions in sync.

Reply:
@thomaseizinger I think my last push answers that :) tell me if it's ok for you